llvm/lib/Target/WebAssembly/AsmParser/WebAssemblyAsmParser.cpp

```diff
@@ -668,6 +668,8 @@ class WebAssemblyAsmParser final : public MCTargetAsmParser {
       if (parseFunctionTableOperand(&FunctionTable))
         return true;
       ExpectFuncType = true;
+    } else if (Name == "ref.test") {
+      ExpectFuncType = true;
```

Comment:
In the long run I guess this should instead be something like `ExpectRefType`?

Reply:
Yeah exactly. But there is no such thing yet...